h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -1485,10 +1485,10 @@ public int compare(Integer x, Integer y) {
   }
 
   public static double [][] convertTo2DMatrix(double [] x, int N) {
-    assert x.length % N == 0;
+    assert x.length % N == 0; // N is number of coefficients per class
```

Comment:
Again assert message please. We will immediately know what is wrong if the code starts failing.